### PR TITLE
Change the code language to English

### PR DESCRIPTION
**Pull request by abcdefg30**
Supersedes #11.

### CODE REVIEW

**Review by abc013 — COMMENTED** [review]
sry, wrong button xD

File: src/blueblock/Human.java [hunk 1]
```diff
@@ -11,29 +11,29 @@
 			new Color(155, 0, 155) };
 	private Label[][] labels = Main.labels;
 	private int pos1, pos2, Dpos1, Dpos2;
-	public int Kills, schritte, SuperScore, PowerUps, Spieler;
-	private boolean lives, poisoned, gesch�tzt;
+	public int Kills, Steps, SuperScore, PowerUps, Player;
+	private boolean lives, poisoned, secured;
```
Comment (abc013) [inline src/blueblock/Human.java]: we should think about another name. This name doesn't exactly explain what it is for.
Why not `armored`?

File: settings.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
-H�he: 8
-Breite: 16
-Spieler: 4
+Height: 8
+Width: 16
+Players: 4
 PowerUps: 2
-Mouse: true
-Kill: false
+Mouse: false
```
Comment (abc013) [inline settings.txt]: `Mouse:` should be `true` as Default.

**Review by abc013 — COMMENTED** [review]
Ok, next try 😆

**Comment by abcdefg30** [comment]
Updated.

**Pull request merged**